client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -65,9 +66,10 @@ export default class ProductFormDetailsCard extends Component {
 					</FormLabel>
 				</div>
 				<div className="products__product-form-details-wrapper">
-					<div className="products__product-form-details-images">
-
-					</div>
+					<ProductFormImages
+						product={ product }
+						editProduct={ editProduct }
+					/>
```

Comment:
Since uploading can be slow, consider allowing ProductFormImages to disable form submittal until upload is complete.